src/constants.js

```diff
@@ -160,6 +160,7 @@ export const PERSISTED_STATE_KEYS = [
   'todaysStartingInventory',
   'toolLevels',
   'useAlternateEndDayButtonPosition',
+  'hideHomeScreen',
```

Comment:
Yay for persisting this setting! Just for the sake of clarity, this property name should be updated to `showHomeScreen` as well.

Reply:
Done :)